examples/java/src/test/java/org/apache/beam/examples/complete/game/UserScoreTest.java

```diff
@@ -92,13 +104,12 @@
   /** Test the {@link ParseEventFn} {@link org.apache.beam.sdk.transforms.DoFn}. */
   @Test
   public void testParseEventFn() throws Exception {
-    DoFnTester<String, GameActionInfo> parseEventFn = DoFnTester.of(new ParseEventFn());
+    PCollection<String> input = p.apply(Create.of(GAME_EVENTS).withCoder(StringUtf8Coder.of()));
```

Comment:
I don't think we need to add explicitly default coder `StringUtf8Coder`

Reply:
I originally had it because the other tests were also using it. I've removed the coder from the other examples to stay consistent.